pkg/kube/translator.go

```diff
@@ -120,17 +136,42 @@ func (t *translator) TranslateUpstreamConfig(au *configv1.ApisixUpstreamConfig)
 }
 
 func (t *translator) TranslateUpstream(namespace, name string, port int32) (*apisixv1.Upstream, error) {
-	endpoints, err := t.EndpointsLister.Endpoints(namespace).Get(name)
-	if err != nil {
+	var endpoints *corev1.Endpoints
+	var endpointSlice *discoveryvibeta1.EndpointSlice
+	var nodes []apisixv1.UpstreamNode
+	var err error
```

Comment:
Use `var` block instead of multiple single line `var` statements.
```suggestion
	var (
	    endpoints *corev1.Endpoints
	    endpointSlice *discoveryvibeta1.EndpointSlice
	    nodes []apisixv1.UpstreamNode
	    err error
	)
```